.github/workflows/workflow-build-docs.yaml

```diff
@@ -25,9 +30,11 @@ jobs:
           cache-dependency-path: "./docs/package-lock.json"
       - run: npm install
       - run: npm run build
-      - name: Deploy
+      - name: Upload artifact
         if: inputs.publish == true
-        uses: peaceiris/actions-gh-pages@v4
+        uses: actions/upload-pages-artifact@v3
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs/dist
+          path: ./docs/dist
+      - name: Deploy to GitHub Pages
```

Comment:
![grafik](https://github.com/it-at-m/digiwf-core/assets/11695964/227877b3-d7b3-4ee2-b0b4-20c1e5343fa5)

wenn du hier noch `environment` hinzufügst, wird im Schritt noch die tolle URL angezeigt.

Reply:
Geht in dem Fall meiner Meinung nach leider nicht, weil wir die CI in jedem PR ausführen und nicht nur auf main.

Reply:
Das ist egal. Ob es auf einem Branch, manuell oder sonst wie ausgeführt wird